NotificationApi/NotificationApi.IntegrationTests/appsettings.json

```diff
@@ -12,7 +12,7 @@
     "TenantId": "TenantId"
   },
   "ConnectionStrings": {
-    "VhNotificationsApi": "Server=(LocalDb)\\MSSQLLocalDB;Database=VhNotificationApi;Trusted_Connection=True;"
+    "VhNotificationsApi": "Server=(LocalDb)\\MSSQLLocalDB;Database=VhNotificationsApi;Trusted_Connection=True;"
```

Comment:
does this match up with the connection string in the API app settings?

Reply:
I changed this to match up with the connection string in the API app settings